bitnami/kafka/templates/kafka-metrics-serviceaccount.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.metrics.kafka.serviceAccount.create }}

```

There's no sense on creating the SA if we're not enabling Kafka metrics:
```suggestion
{{- if and .Values.metrics.kafka.enabled Values.metrics.kafka.serviceAccount.create  }}
```

---

fixed